# Region of Interest message

# ID of the region
uint32 region_id

# Bounds of the region (Array of Long/Lat)
Coordinate[] bounds

# Rover Path (Array of Long/Lat)
Coordinate[] rover_path

# UAV Path (Array of Long/Lat)
Coordinate[] uav_path
